.github/workflows/workflow.yml

```diff
@@ -106,15 +106,15 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
         if: ${{ !steps.release-check.outputs.IS_RELEASE }}
-      - name: Install npm@7
-        run: npm install -g npm@7
+      - name: Install npm@8 # this is needed for proper npm workspaces support as on 12.20 per default npm 6 is installed
```

Comment:
npm 8 is needed as lower versions are failing due to lacking npm workspaces support

Reply:
What difference does it make since we are using Lerna anyway?

Reply:
install is done with npm workspaces – npm ci will fail with, in npm 8 this is fixed:
(This happens as the lib is not built at this point of time)

<img width="1446" alt="CleanShot 2022-09-26 at 12 43 53@2x" src="https://user-images.githubusercontent.com/11156362/192257669-6b9f9399-82d4-43e1-9cfd-b9a240cc2459.png">